src/manipulation.js

```diff
@@ -23,7 +24,7 @@ define( [
 	"./traversing",
 	"./selector",
```

Comment:
:+1: This one stays, though (manipulation functions use `jQuery.isXMLDoc` and `jQuery.text`).